packages/core/src/common/object.spec.ts

```diff
@@ -0,0 +1,111 @@
+// *****************************************************************************
+// Copyright (C) 2022 Ericsson and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { expect } from 'chai';
+import { deepClone, deepFreeze, isEmpty, notEmpty } from './objects';
+
+describe('Object', () => {
+
+    describe('#deepClone', () => {
+
+        describe('primitives', () => {
+            it('should deep clone numbers', () => {
+                expect(1).to.equal(deepClone(1));
```

Comment:
Do you mind reversing the logic (my bad):

```suggestion
                expect(deepClone(1)).to.equal(1);
```

Can you go over all `expect` to align?